docs/README.md

```diff
@@ -49,7 +49,7 @@ and then launching your hubot.
     Hubot>
 
 This starts hubot using the [shell adapter](adapters/shell.md), which
-is mostly useful for development. Make note of  `Hubot>`; this is the name he'll
+is mostly useful for development. Make note of  `Hubot>`; this is the name hubot will
```

Comment:
I think 'your hubot' might be better here, to have parity with the other updates.